src/Business Foundation/App/NoSeriesCopilot/src/Setup [DRAFT]/NoSeriesCopilotSetup.Table.al

```diff
@@ -0,0 +1,679 @@
+        }
+
+        field(14; "Tool 1 Number Guideline Prompt"; Guid)
+        {
+            DataClassification = CustomerContent;
+            Caption = 'Tool 1 Series Numbering Guideline Prompt';
+        }
+        field(15; "Tool 1 Output Examples Prompt"; Guid)
+        {
+            DataClassification = CustomerContent;
+            Caption = 'Tool 1 Output Examples Prompt';
+        }
+
+        field(16; "Tool 1 Output Format Prompt"; Guid)
+        {
+            DataClassification = CustomerContent;
+            Caption = 'Tool 1 Output Format Prompt';
+        }
+        field(17; "Tool 1 Custom Patterns Prompt"; Guid)
+        {
+            DataClassification = CustomerContent;
+            Caption = 'Tool 1 Custom Patterns Prompt';
+        }
+
+        field(19; "Tool 1 Definition"; Guid)
+        {
+            DataClassification = CustomerContent;
+            Caption = 'Tool 1 Definition';
+        }
+
+        field(20; "Tool 2 General Instr. Prompt"; Guid)
+        {
+            DataClassification = CustomerContent;
+            Caption = 'Tool 2 General Instructions Prompt';
+        }
+        field(21; "Tool 2 Limitations Prompt"; Guid)
+        {
+            DataClassification = CustomerContent;
+            Caption = 'Tool 2 Limitations Prompt';
+        }
+        field(22; "Tool 2 Code Guideline Prompt"; Guid)
+        {
+            DataClassification = CustomerContent;
+            Caption = 'Tool 2 Series Code Guideline Prompt';
+        }
+        field(23; "Tool 2 Descr. Guideline Prompt"; Guid)
+        {
+            DataClassification = CustomerContent;
+            Caption = 'Tool 2 Series Description Guideline Prompt';
+        }
+        field(24; "Tool 2 Number Guideline Prompt"; Guid)
+        {
+            DataClassification = CustomerContent;
+            Caption = 'Tool 2 Series Numbering Guideline Prompt';
+        }
+        field(25; "Tool 2 Output Examples Prompt"; Guid)
+        {
+            DataClassification = CustomerContent;
+            Caption = 'Tool 2 Output Examples Prompt';
+        }
+        field(26; "Tool 2 Output Format Prompt"; Guid)
+        {
+            DataClassification = CustomerContent;
+            Caption = 'Tool 2 Output Format Prompt';
+        }
+        field(27; "Tool 2 Custom Patterns Prompt"; Guid)
+        {
+            DataClassification = CustomerContent;
+            Caption = 'Tool 2 Custom Patterns Prompt';
+        }
+        field(29; "Tool 2 Definition"; Guid)
+        {
+            DataClassification = CustomerContent;
+            Caption = 'Tool 2 Definition';
+        }
+
+        field(100; "No. Series Gen. System Prompt"; Guid)
+        {
+            DataClassification = CustomerContent;
+            Caption = 'No. Series Generation System Prompt';
+        }
+    }
+
+    keys
+    {
+        key(PK; Code)
+        {
+            Clustered = true;
+        }
+    }
+
+    procedure GetEndpoint() Endpoint: Text[250]
```

Comment:
All these methods should be add to a codeunit somewhere. Logic should reside in codeunits not tables, pages etc...

I get the appeal of these getters but they are not used consistently. Sometimes they are used, sometimes not. They are also not using the Rec. as I'd expect but instead do their own Get... yes, these will likely hit the cache but still, we have Setup.Get all over the place now instead of getting the record once only..